sdk/docs/manually-written/sdk/tutorials/setup-assistant/installation.rst

```diff
@@ -136,6 +136,36 @@ Download Manually
 
 If you want to verify the SDK download for security purposes before installing, you can look at :doc:`our detailed instructions for manual download and installation <manual-download>`.
 
+Troubleshooting
+***************
+
+The Daml installation requires around 1.5 GB of disk space. If there is not enough space on disk, the installer outputs the error:
+``
+resource exhausted (No space left on device)
+``
+
+In order to fix this problem, one needs to ensure sufficient disk space overall, and in particuloar that the user's home directory and the temporary
+files directory have sufficient space (i.e., in the order of 2GiB each):
+
+* On Windows, one can find the user's home directory in the `%USERPROFILE%` environment variable, and the temporary files directory in the
```

Comment:
I think we need to mention DAML_HOME instead of HOME here, since HOME is only partially relevant for where the SDK installs.

According to our code:

```
-- | The DAML_HOME environment variable determines the path of the daml
-- assistant data directory. This defaults to:
--
--     System.Directory.getAppUserDataDirectory "daml"
--
-- On Linux and Mac, that's ~/.daml
-- On Windows, that's %APPDATA%/daml
```

So we could tell the user that DAML_HOME sets where Daml SDK is installed, and that it defaults to the locations listed above.